azure-pipelines.yml

```diff
@@ -96,6 +96,22 @@ jobs:
   - bash: ci/scripts/check-vendoring.sh
     displayName: Check vendored directories are up-to-date
 
+- job: verible_format
```

Comment:
Can you put the contents of this job into the lint job above by creating a script similar to the ones that we have currently?

Reply:
Done, I also updated the `quick-lint.sh` to run the same script.